components/App.jsx

```diff
@@ -70,13 +71,15 @@ class App extends Component {
98352945401,0],[5.13302083115944,52.34666303592526,0],[5.132935812647075,52.34706328004633,0],[5.13355402129489,52.34745793248898,0],[5.134240352818658,52.34751762283858,0],[5.1317502001379705,52.36378936404806,0],[5.1313847253995615,52.365164417475285,0],[5.130142097909352,52.36701322921324,0],[5.123488052031104,52.37374824843586,0],[5.122527565607362,52.37507659457437,0],[5.122197933832412,52.37613311377432,0],[5.122158946305522,52.376892307776814,0],[5.122579987378232,52.378509371584315,0],[5.123211722825473,52.379470440399935,0],[5.124299395029274,52.38053796978938,0],[5.1256471987172345,52.38133187919157,0],[5.128195187362092,52.38239994517015,0],[5.128563293884731,52.38280148785258,0],[5.127236483518812,52.38429079201789,0],[5.128683462210057,52.38287556518299,0],[5.128941855995542,52.382886680213666,0],[5.141107195776684,52.38735228662374,0],[5.1398148816239715,52.38888925595412,0],[5.141386597630596,52.387463593550365,0],[5.1566634532570035,52.39307030520663,0],[5.1575468912370255,52.393862943649715,0],[5.161878404954117,52.39951315731097,0],[5.125446249689139,52.38618770520863,0],[5.121345427409979,52.38471800149274,0],[5.121233019941465,52.384786288480704,0],[5.16145176933152,52.39952288773215,0],[5.16160546153238,52.399802639087554,0],[5.16811605345847,52.402213736248896,0],[5.208095574822137,52.41681114322412,0],[5.208390062118387,52.416936136506955,0],[5.208242944031195,52.41719017089383,0],[5.208689152447713,52.417312812922304,0],[5.208911856219618,52.41712675628589,0],[5.210903710917563,52.417844384755604,0],[5.2108016101167305,52.418088180530546,0],[5.211146644816041,52.418221834150465,0],[5.211431148444761,52.41803328515748,0],[5.226683620071375,52.42359518931414,0],[5.2271780917237125,52.42361561209175,0],[5.227293958533288,52.42343023078197,0],[5.227176267744256,52.4233390288121,0],[5.208256258451061,52.416368032463225,0],[5.216968011070654,52.416433240199005,0],[5.220508690989157,52.41771812645299,0],[5.2200204059995405,52.41831269876899,0],[5.220171918355824,52.41836960541669,0],[5.222115058719328,52.41789989911589,0],[5.222920200786702,52.417255517477585,0],[5.222252030284466,52.41794616862909,0],[5.222423171504107,52.418009975138496,0],[5.222323733655691,52.41830960580061,0],[5.221776180668316,52.41897969663982,0],[5.2220120659588325,52.41901095146083,0],[5.222685939175432,52.4183693814282,0],[5.223952345674275,52.417795635535285,0],[5.224594869853208,52.41802586130106,0],[5.2244007991520025,52.41884933473839,0],[5.224571001959042,52.41917511235546,0],[5.228473928227682,52.42062947296843,0],[5.2327479662918,52.42531473777085,0],[5.2302192354238,52.42445074034533,0],[5.229840912479666,52.424532779964416,0],[5.229925841160608,52.42477761274324,0],[5.253723174712094,52.433467538643136,0],[5.258424253672872,52.43542550450104,0],[5.26658219754213,52.42840549619775,0],[5.268977396558722,52.426702984426996,0],[5.27116254173139,52.425423528452164,0]]]},"properties":{"name":"Almere","type":"MUNICIPALITY","area":373296142.8079373}}')
```

Comment:
wellicht is het wat om deze extern te houden en require('./bla.json') te doen? dan maakt deze regel je file niet zo fugly